_docs/getting-started-with-wdl.md

```diff
@@ -109,16 +109,21 @@ runtime {
 }
 ```
 
-Finally, we have a metadata section where we can store key value pairs. It is freeform, so we could put anything here. Dockstore is able to pick up both Author and Email if they are defined here. In this example we only have the Author.
+Finally, we have a metadata section where we can store key value pairs. It is freeform, so we could put anything here. Dockstore is able to pick up author, email, and description if they are defined here. 
+The description field can be used to add documentation to your workflow and can make use of markdown syntax. Currently, description values are limited to single line string, but multi-line documentation is possible by including new line characters into the string. 
+Below we have an example of a metadata section with author, email, and a multi-line description.
 
 ```
 meta {
     author: "Andrew Duncan"
+    email: "andrew@foobar.com"
+    description: "This is my WDL workflow. \n\n Adding documentation to my workflow improves clarity"
```

Comment:
Okay I think it would also help to include an image of how the code will display:


> ...  <br> Below we show an example meta section and how it will display on your workflow's landing page:

```
meta {
	author: "Andrew Duncan"
        email: "andrew@foobar.com"
        description: "## Bamstats \n This is the Bamstats workflow.\n\n Adding documentation improves clarity."
}
```

<img width="736" alt="wdl_meta_example" src="https://user-images.githubusercontent.com/23464754/52145429-c56e0000-2615-11e9-8255-a10510aa78f6.png">

Reply:
I like the screenshot!